Libraries/Image/RCTImageView.m

```diff
@@ -144,7 +144,19 @@ - (void)reloadImage
         }
       } else {
         if (_onLoad) {
-          NSDictionary *event = @{ @"target": self.reactTag };
+          CGSize intrinsicSize = [dimensions CGSizeValue];
+          NSDictionary *event;
```

Comment:
In terms of code style it might be more succinct to start with 
```
[NSMutableDictionary dictionaryWithDictionary:@{
  @"target": self.reactTag,
}];
```
and add "size" to it later